Description: mips64 is not properly detected as 64-bits
Upstream: no
diff --git a/src/include/janet.h b/src/include/janet.h
index 4448a90..dc73001 100644
--- a/src/include/janet.h
+++ b/src/include/janet.h
@@ -96,7 +96,8 @@ extern "C" {
     || (defined(__sparc__) && defined(__arch64__) || defined (__sparcv9)) /* BE */ \
     || defined(__s390x__) /* S390 64-bit (BE) */ \
     || (defined(__ppc64__) || defined(__PPC64__)) \
-    || defined(__aarch64__) /* ARM 64-bit */
+    || defined(__aarch64__) /* ARM 64-bit */ \
+    || defined(__mips64) /* MIPS 64-bit */
 #define JANET_64 1
 #else
 #define JANET_32 1
